dvc/stage/__init__.py

```diff
@@ -36,16 +49,24 @@
 
 if TYPE_CHECKING:
     from dvc.dvcfile import DVCFile
+    from dvc.output import Output
+    from dvc.repo import Repo
+    from dvc.types import StrPath
     from dvc_data.hashfile.hash_info import HashInfo
     from dvc_objects.db.base import ObjectDB
 
 logger = logging.getLogger(__name__)
 # Disallow all punctuation characters except hyphen and underscore
 INVALID_STAGENAME_CHARS = set(string.punctuation) - {"_", "-"}
 Env = Dict[str, str]
+ChangedEntries = Tuple[List[str], List[str], Optional[str]]
+
+_T = TypeVar("_T")
```

Comment:
Is this some sort of convention?

Reply:
You mean `TypeVar("_T")`? I guess the main point is to keep it "private"